packages/terra-graphs-docs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cerner/terra-graphs-docs",
-  "version": "0.0.0",
+  "version": "0.0.1",
```

Comment:
Do we want to make this version 1.0.0?

Reply:
Not so sure about this, it is the version that is added by Ryan initially as 0.0.0 so I left it as is and it got updated to 0.0.1? 
https://github.com/cerner/terra-graphs/pull/246/files#diff-d761c3fb8ba076a072500ff322d01b361058688b30dff1e2cd8fd703d36b6da2R3

Reply:
If terra-graph-docs is ready to go, the initial official release should be 1.0.0.

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/274/commits/81caeac55d11d0c6e5a2d9d6a8a544a65aaa6c2f#diff-d761c3fb8ba076a072500ff322d01b361058688b30dff1e2cd8fd703d36b6da2R3).